test/functionalTest/cases/1289_exceed_mongo_key_limit/exceed_mongo_key_limit.test

```diff
@@ -0,0 +1,86 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
what if I create a folder named  1289_exceed_mongo_key_limit_v2 for this test and use the current folder for the V1 test